src/images/icons/codeBlock.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="17" height="17" viewBox="0 0 17 17" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
A question, in the Dapp we always tried to use icons with a viewbox of `0 0 30 30`, has that changed in the CDapp? In the icons added in this PR I see that there are a few with `0 0 20 20` and others like this one that are different.

Reply:
These are taken from the dapp. I think if it's important, we should create a separate issue for enforcing the same viewbox on all svgs in the repo as there are a few exceptions that've been ported over from the dapp. 

What's the rationale for wanting them all to be `0 0 30 30`, btw?

Reply:
As far as I remember, the reason for all of them to be like that was to make sure they are all consistently formatted the same way